spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -126,4 +126,34 @@
       end
     end
   end
+
+  describe 'GET#show' do
+    context 'given a valid survey ID' do
+      it 'returns success status' do
+        user = api_sign_in
+        Fabricate(:keyword, user: user, id: 1)
+        get :show, params: { id: 1 }
+
+        expect(response).to have_http_status(:success)
+      end
+
+      it 'returns the expected keyword' do
+        user = api_sign_in
+        Fabricate(:keyword, user: user, id: 1)
+        get :show, params: { id: 1 }
+        body = JSON.parse(response.body, symbolize_names: true)
+
+        expect(body[:data][:id]).to eq('1')
+      end
+    end
+
+    context 'given an invalid survey ID' do
```

Comment:
```suggestion
    context 'given an invalid keyword ID' do
```

Reply:
Fixed in [`94a0f67` (#56)](https://github.com/suho/google-search-ruby/pull/56/commits/94a0f67a4744e5efbb7f23d7b1f076b5e191f3c5)